cosmos/constants.py

```diff
@@ -14,6 +14,7 @@
 DBT_PARTIAL_PARSE_FILE_NAME = "partial_parse.msgpack"
 DBT_LOG_FILENAME = "dbt.log"
 DBT_BINARY_NAME = "dbt"
+DBT_PACKAGES_CONFIG_PATH = Path(os.path.expanduser("~")).joinpath(".dbt/packages.yml")
```

Comment:
Isn't the `packages.yml` relative to the dbt project, which may be in any path? It's fine to have in constants only `packages.yml`, similar to `DBT_PARTIAL_PARSE_FILE_NAME` and build the full path where we need it